lib/read_aligner.cc

```diff
@@ -532,12 +532,13 @@ Alignment* ReadAligner::Align(const std::string& read)
     }
 
     HashIntoType fhash = 0, rhash = 0;
-    _hash(start.kmer, k, fhash, rhash);
+    fhash = m_ch->hash_dna_top_strand(start.kmer.c_str());
+    rhash = m_ch->hash_dna_bottom_strand(start.kmer.c_str());
```

Comment:
Take a look at http://cpp.sh/5fce6 for something akin to "tuple unpacking" for return values. Might not be strictly necessary here but IMO makes for nice code especially compared to (ab)using arguments for return values 😃